src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -5,6 +5,27 @@
 - Fixed `@udf` decorator when `packages=None` are specified preventing error in `Series.apply`.
 - Fixed incorrect regex used in `Series.str.contains`.
 
+### Behavior Changes
+- Raise not implemented error instead of fallback to pandas in following APIs:
+  - `pd.merge`, `DataFrame.merge` and `DataFrame.join` if given the `validate` parameter.
+  - `pd.to_numeric` if `error == 'ignore'`.
+  - `pd.to_datetime` if `format` is None or not supported in Snowflake or if `exact`, `infer_datetime_format` parameters are given or `origin == 'julian'` or `error == 'ignore'`.
+  - `DataFrame/Series.all` if called on non-integer/boolean columns.
+  - `DataFrame/Series.any` if called on non-integer/boolean columns.
+  - `DataFrame/Series.astype` if casting from string to datetime or `errors == 'ignore'`.
+  - `DataFrame/Series.dropna` if `axis == 0`
```

Comment:
Should this be `DataFrame/Series.dropna` if `axis == 1`?
The [docs](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/supported/dataframe_supported.html) for dropna reads `D if axis == 1`

Reply:
Good catch, it should be axis == 1